cpg_pipes/types.py

```diff
@@ -141,36 +141,6 @@ def __str__(self):
 FastqPairs = List[FastqPair]
 
 
-# Alignment input can be a CRAM file on a bucket, or a list of Fastq pairs on a bucket.
-AlignmentInput = Union[FastqPairs, CramPath]
-
-
-def alignment_input_exists(v: AlignmentInput) -> bool:
-    """
-    Check if all files in alignment inputs exist.
-    """
-    if isinstance(v, CramPath):
-        return utils.exists(v.path)
-    else:
-        return all(utils.exists(pair.r1) and utils.exists(pair.r2) for pair in v)
-
-
-def alignment_input_short_str(v: AlignmentInput) -> str:
-    """
-    For CRAM, it's just a path to CRAM. For FASTQ pairs, it's a FASTQ path glob.
-    """
-    if isinstance(v, CramPath):
-        return str(v.path)
-    else:
-        all_fastq_paths = []
-        for pair in v:
-            all_fastq_paths.extend([pair.r1, pair.r2])
-        return ''.join([
-            f'{{{",".join(set(chars))}}}' if len(set(chars)) > 1 else chars[0] 
-            for chars in zip(*map(str, all_fastq_paths))
-        ])
-
-
 class SequencingType(Enum):
     """
     Type (scope) of a sequencing experiment.
```

Comment:
You wouldn't consider extending the SequencingType from the SM API? I'm just about to add an ONT sequencing type, and we don't expect any refactoring of these values from now.

Reply:
It would imply a hard dependency on the `sample-metadata` package. I guess why not though

Reply:
Ah, actually the enum is not a part of the `sample-metadata` package. It's in the `models` folder and `models` is not packaged.